INDLinkLabel.swift

```diff
@@ -189,7 +188,7 @@ import UIKit
             // needs to be unrestricted for it to correctly lay out all the text.
             // Might be due to a difference in the computed text sizes of `UILabel`
             // and `NSLayoutManager`.
-            textContainer.size = CGSize(width: CGRectGetWidth(bounds), height: CGFloat.max)
+            textContainer.size = self.textRectForBounds(self.bounds, limitedToNumberOfLines: self.numberOfLines).size
```

Comment:
This is what I had earlier, but it didn't work correctly (see the comment above this line). It seems that `NSTextContainer` requires more room than `-textRectForBounds:limitedToNumberOfLines` returns to correctly lay out all of the text. This also doesn't solve the issue with the text insets completely because the y-origin of the text rect isn't taken into account. Tapped points on the view would need to be adjusted using the y-origin before passing them to the layout manager.